pkg/apis/servicecatalog/validation/binding.go

```diff
@@ -120,6 +120,10 @@ func validateServiceBindingStatus(status *sc.ServiceBindingStatus, fldPath *fiel
 	}
 
 	if status.InProgressProperties != nil {
+		if status.ExternalProperties != nil {
+			allErrs = append(allErrs, field.Forbidden(fldPath.Child("inProgressProperties"), `inProgressProperties and externalProperties cannot both be present`))
```

Comment:
I would like to hold off on this until we get an agreement in #1546 about whether the `ExternalProperties` should be set as soon as a successful response to a bind request is received.